src/Notification/Model/Notification.php

```diff
@@ -39,6 +50,7 @@ class Notification extends Model
         'title'    => 'string',
         'content'  => 'string',
         'category' => 'string',
+        'tags'     => NotificationTags::class,
```

Comment:
Ik denk dat dit beter (of net zo goed) een simpel object of array kan zijn. Het voelt een beetje als overkill, maar ook rigide om er een heel model voor te maken. Nu moet je dat steeds gaan uitbreiden als er ooit eens andere tags bij komen kijken, terwijl het ook prima een beetje een "black box" kan zijn, dat je de frontend laat kijken wat het ermee doet